src/simmi/TestCase_17/TestCase17.java

```diff
@@ -0,0 +1,89 @@
+package simmi.TestCase_17;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/*Test Case 17: 
+Today's assignment (25/6/2021)
+www.flipcart.com
+1. Hover on Fashion icon
+2. Hover on the first  item>> count the number of elements display on hovering.
+*/
+
+public class TestCase17 {
+
+	@Test
+	static ArrayList<String> getElement(WebDriver driver) {
+		WebElement target = driver.findElement(By.xpath("//img[@alt='Fashion']"));
+
+		Actions action = new Actions(driver);
+		action.moveToElement(target).build().perform();
+
+		List<WebElement> e = driver.findElements(By.xpath("//div[@class='_3XS_gI _7qr1OC']/a"));
+		int count = driver.findElements(By.xpath("//div[@class='_3XS_gI _7qr1OC']/a")).size();
+		Assert.assertEquals(count, 11);
+		System.out.println("Total numbers of Elements after 1st hover on Fashion is " + count);
+
+		Assert.assertEquals(driver.findElements(By.xpath("//div[@class='_3XS_gI _7qr1OC']/a")).size(), 11);
+		ArrayList<String> output = new ArrayList<String>();
+
+		for (WebElement elementList : e) {
+			if (e.size() <= 11) {
+				output.add(elementList.getText());
+			}
+		}
+
+		return output;
+	}
+
+	@Test
```

Comment:
this @Test should be applied on service method, you are calling this method the operation so.
Remember this point for future references.